app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -63,4 +70,6 @@ function DocumentsStep({
   );
 }
 
-export default withBackgroundEncryptedUpload(DocumentsStep);
+const DocumentsStepWithBackgroundUpload = withBackgroundEncryptedUpload(DocumentsStep);
+
+export { DocumentsStepWithBackgroundUpload as DocumentsStep, documentsStepValidator };
```

Comment:
Not sure if you had a preference to have these both be named exports, but it's valid to have both default and named exports, so the step could remain as the default export.

```suggestion
export default withBackgroundEncryptedUpload(DocumentsStep);

export { documentsStepValidator };
```

Elsewhere, to import:

```js
import DocumentsStep, { documentsStepValidator } from './documents-step';
```

Reply:
Oh this is way clearer, thanks! I had a ton of issues with the renaming export syntax, this definitely seems cleaner